src/directives/profile.js

```diff
@@ -65,6 +93,12 @@ ngeo.profileDirective = function() {
               profile.showPois(poiData);
             }
           }
+
+          scope.$watch(attrs['ngeoProfileHighlight'], function(newVal, oldVal) {
+            if (goog.isDef(profile) && goog.isDef(elevationData)) {
+              profile.highlight(newVal);
+            }
+          });
```

Comment:
This piece should be removed. It's a remaining code from when I was experimenting with a `highlight` directive attribute.